source/features/view-markdown-source.tsx

```diff
@@ -53,6 +119,7 @@ async function showSource(): Promise<void> {
 	sourceButton.classList.add('selected');
 	renderedButton.classList.remove('selected');
 	blurButton(sourceButton);
+	select('.blob-wrapper')!.before(lineActions());
```

Comment:
We already have this element, I think it's
```suggestion
	(await source).before(lineActions());
```

Also, what happens if I toggle between source and rendered multiple times? It looks like it's just going to add it multiple times.

If you wrap `lineActions` with `onetime`, you can add this in `showRendered` I think:

```js
lineActions().remove();
```

You can find this pattern in other parts of RG